app/processing/delta/assign-ledger/split-to-ledger.js

```diff
@@ -9,7 +9,7 @@ const splitToLedger = (paymentRequest, targetValue, ledger) => {
   const originalValue = paymentRequest.value
   const updatedValue = ledger === AP ? originalValue + targetValue : originalValue - targetValue
 
-  paymentRequest.originalInvoiceNumber = paymentRequest.invoiceNumber
+  paymentRequest.originalInvoiceNumber = paymentRequest.originalInvoiceNumber ?? paymentRequest.invoiceNumber
```

Comment:
Why does this need to change?

Reply:
Oh I see.  Because if it's there you don't want to overwrite it.

Makes sense, however a manual invoice doesn't hit any of this logic so `originalInvoiceNumber` will never have a value.

Suggest leaving this as it was.

Reply:
reverted as per suggestion